docs/regenerate_docs.sh

```diff
@@ -19,9 +19,9 @@
 
 set -euo pipefail
 
-bazel build docs:all --experimental_remap_main_repo
+bazel build docs:all
 
-for filename in bazel-bin/docs/*_gen.md; do
```

Comment:
I would prefer to avoid changing the rename logic here (and thus avoid changing all the `out`s in docs/BUILD) without a specific reason. The existing logic works, and "_gen" suffix in a generated doc file is a good signal.